xml/depl_nodes.xml

```diff
@@ -996,14 +996,36 @@ getent group postgres | cut -d: -f3</screen>
    <varlistentry>
     <term>
      <guimenu>
-      Default Credentials: Regular User/Administrator User Name/Password
+      Default Credentials: Administrator User Name/Password
      </guimenu>
     </term>
     <listitem>
      <para>
-      User name and password for the regular user and the administrator.
-      Both accounts can be used to log in to the &cloud; &dash; to
-      manage &o_ident; users and access.
+      User name and password for the administrator.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <guimenu>
+      Default Credentials: Create Regular User
+     </guimenu>
+    </term>
+    <listitem>
+     <para>
+      Enable or disable the ability to create the regular user.
```

Comment:
its not whether you have the *ability* to create a regular user, it is about whether a default user (with a default password) is created *automatically* (which you normally don't want, especially in a LDAP environment where you're operating on possibly read only identity information).